### DISCUSSION TITLE
Using Annotated with Pydantic's Field()

### DISCUSSION THREAD

**Discussion by deniselupe** [fastapi/fastapi#9446] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# Example from current Body - Fields documentation

from typing import Union

from fastapi import Body, FastAPI
from pydantic import BaseModel, Field
from typing_extensions import Annotated

app = FastAPI()


class Item(BaseModel):
    name: str
    description: Union[str, None] = Field(
        default=None, title="The description of the item", max_length=300
    )
    price: float = Field(gt=0, description="The price must be greater than zero")
    tax: Union[float, None] = None


@app.put("/items/{item_id}")
async def update_item(item_id: int, item: Annotated[Item, Body(embed=True)]):
    results = {"item_id": item_id, "item": item}
    return results
```

### Description

Hello, sorry if this is the wrong place to ask as I'm still new to FastAPI.

I see that the current [Body - Fields](https://fastapi.tiangolo.com/tutorial/body-fields/#import-field) documentation was updated with examples that use `Annotated`, however, I see that within this documentation Pydantic's `Field` is still being used in the same manner as FastAPI <0.95.1.

I rewrote the code so that `Field()` was placed inside `Annotated`, and so far I have not experienced any differences between the behavior, response body, or OpenAPI documentation of both models.

```
from typing import Union
from fastapi import Body, FastAPI 
from pydantic import BaseModel, Field
from typing_extensions import Annotated

app = FastAPI()

class Item(BaseModel):
    name: str
    description: Annotated[Union[str, None], Field(title="The description of the item", max_length=300)] = None
    price: Annotated[float, Field(gt=0, description="The price must be greater than zero")]
    tax: Union[float, None] = None

@app.put("/v1/items/{item_id}")
async def update_item(item_id: int, item: Annotated[Item, Body(embed=True)]):
    results = {"item_id": item_id, "item": item}
    return results
```

Is it okay if I continue using `Annotated` with `Field` like in the example above, or should I continue to use `Annotated` for only the `Query`, `Path`, and `Body`?

### Operating System

Linux

### Operating System Details

NAME="Ubuntu"
VERSION="16.04.7 LTS (Xenial Xerus)"

### FastAPI Version

0.95.1

### Python Version

Python 3.8.7

**Comment by nzig** [comment, marked as answer]
I think this is a bug in the documentation, though not necessarily.

Your question is really more about pydantic, since `Item` is just a pydantic model. It has nothing specific to FastAPI about it. Pydantic allows you to use either `Annotated` or ` = Field()`, as documented [here](https://docs.pydantic.dev/latest/usage/schema/#typingannotated-fields).

**Reply by deniselupe** [reply]
Awesome! It seems that I can then move forward with using `Annotated` to keep things consistent. Thank you for referring me to the documentation!
